server/container_create.go

```diff
@@ -932,8 +931,10 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 		return nil, err
 	}
 
-	if err = addSecretsBindMounts(mountLabel, containerInfo.RunDir, s.config.DefaultMountsPath, specgen); err != nil {
-		return nil, fmt.Errorf("failed to mount secrets: %v", err)
+	if s.config.DefaultMountsPath != "" {
```

Comment:
This will always be true because `config.DefaultMountsPath` is set to `DefaultMountsFile` by default in the config. Take that out I guess for this to work.

Reply:
if I set that to "" then we won't add any mounts so this check is needed no?

Reply:
see more here https://github.com/kubernetes-incubator/cri-o/pull/1003#discussion_r144319903

Reply:
yeah, sorry that is what I meant by take that out - remove the default from the config.